Telepathy/Server.cs

```diff
@@ -250,6 +243,39 @@ void Listen(int port)
             }
         }
 
+        bool TryListen(int port)
+        {
+            bool successful = false;
+            try
+            {
+                // start listener
+                listener = new TcpListener(new IPEndPoint(IPAddress.Any, port));
+                listener.Server.NoDelay = NoDelay;
+                listener.Server.SendTimeout = SendTimeout;
+                listener.Start();
+                Logger.Log("Server: listening port=" + port);
+                successful = true;
```

Comment:
Remove successful, put return true here and return false at the end?

Reply:
Sure. I've never really used used try and catch and it seemed weird that the compiler didn't complain if you had a return in the try but no return if the try failed so I did that to be certain. Should I change it or can you edit pull requests?

Reply:
And of course I would have another return at the end, but again, I thought it was weird that the compiler didn't care.

Reply:
Fixed.